android-exoplayer/src/main/java/com/brentvatne/exoplayer/ReactExoplayerView.java

```diff
@@ -257,6 +261,41 @@ public void onBandwidthSample(int elapsedMs, long bytes, long bitrate) {
     }
 
     // Internal methods
+
+    /**
+     * Toggling the visibility of the player control view 
+     */
+    private void togglePlayerControlVisibility() {
+        if(playerControlView.isVisible()) {
+            playerControlView.setVisibility(INVISIBLE);
+        } else {
+            playerControlView.setVisibility(VISIBLE);
+        }
+    }
+
+    /**
+     * Initialising Player control
+     */
+    private void initialisePlayerControl() {
```

Comment:
Minor typo, the correct spelling is `initialize`.

Reply:
Updated **initialise** as **initialize**